torch/csrc/distributed/rpc/rref_context.h

```diff
@@ -31,54 +31,54 @@ c10::intrusive_ptr<RRef> TORCH_API finishCreatingOwnerRRef(
 using torch::utils::Future;
 
 // Manages RRef lifetime and keeps track of RRef forks.
-class TORCH_API RRefContext {
+class RRefContext {
```

Comment:
Let's add a comment to explain why `TORCH_API` needs to be moved to individual functions.

Reply:
Since TORCH_API is defined as **__declspec(dllimport)** by default. thread_local variable in MSVC does allow add a __declspec(dllimport).

Reply:
Same as above, please use C10_EXPORT and create TORCH_PYTHON_API macro